app/src/main/java/de/westnordost/streetcomplete/screens/main/edithistory/UndoDialog.kt

```diff
@@ -151,18 +152,30 @@ class UndoDialog(
            "<li>" +
            context.resources.getString(
                change.titleResId,
-               "<tt>" + Html.escapeHtml(change.tagString) + "</tt>"
+               "<tt>" + change.tagString() + "</tt>"
            ) +
            "</li>"
         })
         return txt
     }
 }
 
-private val StringMapEntryChange.tagString: String get() = when (this) {
-    is StringMapEntryAdd -> "$key = $value"
-    is StringMapEntryModify -> "$key = $value"
-    is StringMapEntryDelete -> "$key = $valueBefore"
+fun StringMapEntryChange.tagString(): String {
+    val keyText = key
+    val escapedKey = Html.escapeHtml(keyText)
+    val valueText = when (this) {
+        is StringMapEntryAdd -> value
+        is StringMapEntryModify -> value
+        is StringMapEntryDelete -> valueBefore
+    }
+    val escapedValue = Html.escapeHtml(valueText)
+    val shownKey = "<a href=\"https://wiki.openstreetmap.org/wiki/Key:$escapedKey\">$escapedKey</a>"
```

Comment:
What if that key is not documented? Surely some keys that StreetComplete uses are not documented? Maybe?

Reply:
> What if that key is not documented? Surely some keys that StreetComplete uses are not documented? Maybe?

Every single key used by StreetComplete is documented. I verified that as part of doing that taginfo listing (and created some tag pages and or improved their documentation).

That took some time, but was worth doing anyway (for example OSM Wiki pages about surfaces are now better matching both reality, OSM mapping consensus and StreetComplete).

Some **removed** tags may not have wiki pages - I think it is fine but I can research also this.

Reply:
And if new tags would become used by SC without having own wiki page then I plan to fix this. Likely by creating stub pages, maybe by changing SC where applicable like https://github.com/streetcomplete/countrymetadata/commit/f9ec20a28a1dab6fb3f877e8891a7ef322ddb68c (I found this bug exactly because I was checking whether all possible links are actually existing)